mxd/3.1_MXD_tutorial1_dataexchange.md

```diff
@@ -0,0 +1,181 @@
+
+Alice, as a data consumer, wants to consume data from Bob. Bob, as a data provider, needs to create an asset for Alice. The data asset should have the following properties:
+
+| ID          | 3                                                                                            |
+|-------------|----------------------------------------------------------------------------------------------|
+| Description | Product EDC Demo Asset 3                                                                     |
+| Type        | HttpData                                                                                     |
+| URL         | [https://jsonplaceholder.typicode.com/todos/3](https://jsonplaceholder.typicode.com/todos/3) |
+
+Action (Bob): Create this asset using the following curl command:
+
+```shell
+curl --location 'http://localhost/bob/management/v2/assets' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+    "@context": {},
+    "asset": {
+        "@type": "Asset",
+        "@id": "3", 
+        "properties": {
+            "description": "Product EDC Demo Asset 3"
+        }
+    },
+    "dataAddress": {
+        "@type": "DataAddress",
+        "type": "HttpData",
+        "baseUrl": "https://jsonplaceholder.typicode.com/todos/3"
+    }
+}'
+```
+
+Bob tells Alice, that he created an asset, and she should now be able to request it. In the next step, Alice requests a contract offer catalog. In this catalog, all contract offers for Alice are listed.
+
+Action (Alice): Execute a request using the following curl commands:
+
+```shell
+curl --location 'http://localhost/alice/management/v2/catalog/request' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+    "@context": {},
+    "protocol": "dataspace-protocol-http",
+    "counterPartyAddress": "http://bob-controlplane:8084/api/v1/dsp",
+    "querySpec": {
+        "offset": 0,
+        "limit": 100,
+        "filter": "",
+        "range": {
+            "from": 0,
+            "to": 100
+        },
+        "criterion": ""
+    }
+}'
+```
+
+Let´s see if Alice can see the Asset (ID:3). Can you find it?
+
+As you can see in the response, the data offer "Product EDC Demo Asset 3" (asset ID:3) does not appear. Unfortunately, Alice sees some contract offers but she cannot find the contract offer from Bob.
+
+Alice calls Bob and says she can´t see the asset. Bob remembers that he did not create an access policy. An access policy defines who is allowed to see a data offering.
+
+Action (Bob): Create the access policy using the following curl command:
+
+```shell
+curl --location 'http://localhost/bob/management/v2/policydefinitions' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "@type": "PolicyDefinitionRequestDto",
+  "@id": "3",
```

Comment:
We can update id as `31` so, access policy id will be 31, and contract policy id will be 32.